### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by tanghel**

### CODE REVIEW

File: .github/workflows/test-cli.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  test-cli:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16
+          registry-url: https://registry.npmjs.org/
+
+      - run: npm install -g @multiversx/sdk-json-schema-expander
```
Comment (bogdan-rosianu) [inline .github/workflows/test-cli.yml]: this should be updated

File: commands/cli.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+import { program } from 'commander';
+import { GenerateSchemaCommand } from './schema/generate';
+import { ExpandSchemaCommand } from './schema/expand';
+import { TypesSchemaCommand } from './schema/types';
+
+// Create a new command object for 'config'
```
Comment (bogdan-rosianu) [inline commands/cli.ts]: rotten comment :D 'config' -> 'schema'

File: package.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@multiversx/sdk-nestjs-cli",
+  "version": "1.0.8",
```
Comment (bogdan-rosianu) [inline package.json]: we should start from 1.0.0 when first publishing

File: .github/workflows/test-cli.yml (hunk 1 above)
Comment (tanghel) [reply .github/workflows/test-cli.yml]: deleted, problem solved 😅

File: commands/cli.ts (hunk 2 above)
Comment (tanghel) [reply commands/cli.ts]: done

File: package.json (hunk 3 above)
Comment (tanghel) [reply package.json]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**